scripts/encoding/encoder.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import pandas as pd
+from sklearn.feature_extraction.text import TfidfVectorizer, CountVectorizer
+
+ENCODER_TOOLS = {
+    'tfidf': TfidfVectorizer(),
+    'vect': CountVectorizer()
```

Comment:
Pas la peine d'instancier, on ne sait pas encore quel encodeur on va utiliser
```suggestion
    'tfidf': TfidfVectorizer,
    'vect': CountVectorizer
```

Reply:
Ca n'est pas une modification prioritaire mais elle peut être bienvenue. Attention à ne pas oublier de changer l'appel du vectoriser à la ligne 24

Reply:
L'idée est que la personne faisant le preprocessing puissent avoir le choix de changer d'encodeur pour x ou y raison sur quelques colonnes, etc...

Reply:
Je comprends mais autant avoir un dictionnaire de classes qu'on instancie quand on décide de s'en servir plutôt qu'un dictionnaire d'objets. C'est pas dramatique mais c'est toujours quelques cycles CPU en moins